deployment-templates/.goreleaser.yml

```diff
@@ -3,9 +3,9 @@ archives:
     name_template: '{{ .Binary }}-{{ .Tag }}-{{ .Os }}-{{ .Arch }}'
 before:
   hooks:
-    - make build_provider
+    - make provider
```

Comment:
we only need to run make tfgen before the provider

Reply:
oh, I'll update that, thank you

Reply:
make tfgen creates the schema which is required for goreleaser to make it's build of the provider

Running make provider again would just duplicate the work :)